### PR TITLE
Day dreaming - Adds the (fungal entity) day tripper.

### PR DESCRIPTION
**Pull request by zdlpoppa**
#### Summary
Content "Adds the fungal acid zombie - day tripper"

#### Purpose of change
This addition is primarily meant to expand more on the 'strange fungal effects' side of the mycus, and since certain mushrooms can cause hallucinations (like magic mushrooms), I decided to add a fungal creature that represents this, if not a little more magical.

#### Describe the solution
This is mostly the basis for more hallucinogenic fungal creatures to come; I think the implication of positive/negative effects to the player by more unique fungaloid creatures will add some further confusion as to whether people should or shouldn't dislike this invasive fungus.

#### Describe alternatives you've considered
Possibly alternative names/ways of going around this creature, right now it's more anomalous then I'd like it to be (even for the strange faction it is), toning down the effects it has is something I'm still considering as of now (since the mycus isn't really combat orientated).

#### Testing
From the days of on-and-off testing I've done, it seems nothing breaks due to the changes I've made. however, some JSON changes  made exclusively in github might cause some errors.

#### Additional context
Don't do magic mushrooms, kids!

### CODE REVIEW

**Comment by fairyarmadillo** [comment]
I don't think we should add any hallucination content until our hallucination effects are better. The status is currently annoying and immersion-breaking more than anything, when it ought to be providing an actual challenge by making the player question whether what they're seeing is real.

This would probably mean cutting the frequency by 75% at least, and figuring out some way to ensure that the player isn't made to see monsters their character could not know about. We track kills, so that would be a good start.

Additionally, hallucinating monsters is the lamest way for hallucinations to play out. We should be messing with terrain, hiding traps or placing fake ones, hiding windows, etc.

**Comment by zdlpoppa** [comment]
> I don't think we should add any hallucination content until our hallucination effects are better. The status is currently annoying and immersion-breaking more than anything, when it ought to be providing an actual challenge by making the player question whether what they're seeing is real.
> 
> This would probably mean cutting the frequency by 75% at least, and figuring out some way to ensure that the player isn't made to see monsters their character could not know about. We track kills, so that would be a good start.
> 
> Additionally, hallucinating monsters is the lamest way for hallucinations to play out. We should be messing with terrain, hiding traps or placing fake ones, hiding windows, etc.

This is one of the reasons I didn't include any hallucinations with this effect (also because I didn't really know how), I do agree that hallucinations could be alot more immersive then how they are now.

I'm personally unable to attempt any work on this, just don't have the ability or knowledge to know where to start, but I'll expand on your comment by saying; if going beyond creatures is seemingly too difficult, I'd like a way to specify what types of creatures I want a person to hallucinate, instead of it being a random assortment.

If a few more people agree with your view, I could look at changing the general effect/theme of this monster. I'm still open to plenty of ideas/interpretations other people may have.

**Comment by fairyarmadillo** [comment]
I think the theme is fine and God knows Mycus needs some help, it's just the current hallucination effect I have a problem with. If it's not being used then I don't really have an opinion.

File: data/json/monsters/fungus_zombie.json [hunk 1]
```diff
@@ -582,6 +582,62 @@
...
+    "vision_night": 3,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_zombie" ],
+    "harvest": "zombie_humanoid_mushroom",
+    "special_attacks": [
+      {
+        "type": "spell",
+        "spell_data": { "id": "spread_fungal_acid", "min_level": 1 },
+        "cooldown": 3,
+        "monster_message": "the % spews hallucinogenic gas!"
```
Comment (Uwuewsky) [inline data/json/monsters/fungus_zombie.json:619]: @zdlpoppa should there be %s here?

**Pull request merged**